### DISCUSSION TITLE
Items for v5.1 release

### DISCUSSION THREAD

**Discussion by lmorg** [lmorg/murex#722] [Ideas]
Here is where people can raise requests, proposals, etc for the v5.1 release of Murex. Website and documentation updates can be released outside of this release schedule, so are exempt from this plan.

- [ ] job control bug fixes
- [ ] new parser for syntax highlighting and completion
- [ ] user facing controls for theming syntax highlighting

...feel free to comment with other requests.

Milestone: https://github.com/lmorg/murex/milestone/1

**Comment by orefalo** [comment]
I believe Github has (free) tooling to achieve milestones in a more structured way.

**Reply by lmorg** [reply]
Thanks for the tip. Looks like you cannot have a discussion around a milestone though (unless I've missed something obvious). So we can have the discussion here and a milestone to track the commits
